js/templates/modals/about/about.html

```diff
@@ -0,0 +1,45 @@
+<div class="topControls flex js-closeClickTarget js-btcTicker"></div>
```

Comment:
'js-closeClickTarget's are not needed anymore since we decided not to close the modals on overlay click.